src/editor/active-editor-hooks.tsx

```diff
@@ -28,7 +28,11 @@ export class EditorActions {
    *
    * @param code The code with any required imports.
    */
-  insertCode = (code: string): void => {
+  insertCode = (code: string, parentSlug?: string): void => {
```

Comment:
I'd have been tempted to name it something more general by the time it reaches this method, e.g. source or id. If we supported dnd / insert from elsewhere we'd provide some similar identifier but it may not exactly be a "parent slug". 

Similarly for the drag context below.

Where we have existing param documentation it would be good to maintain it.